app/views/welcome/_next-event.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class='main'>
+  <h1> Next Event </h1>
+  <% if !next_event %>
```

Comment:
It reads a little nicer if we start with the thing that we want as opposed to the fallback:

```ruby
<% if thing_we_want %>

<% else %>

<% end %>
```